vcf/vcf.go

```diff
@@ -122,6 +122,12 @@ func ParseNotes(data string, format []string) []GenomeSample {
 	if format[0] != "GT" && format[0] != "." { //len(format) == 0 checks for when there is info in the notes column but no format specification
 		log.Fatalf("VCF format files with sample information must begin with \"GT\" as the first format column or be marked blank with a period. Here was the first format entry: %s.\nError parsing the line with this Notes column: %s.\n", format[0], data)
 	}
+
+	if format[0] == "." { //if the format column is blank, we do not need to parse further.
```

Comment:
does something break without this check? If so, could you add a test (or at least a comment) for this so it doesn't get removed in the future?

Reply:
We actually do run into another issue here that is unresolved. The VCF file in the gtf package contains a space and then a "." in the format column, and thus can't be read and written with fidelity. We could check for format[0] contains "." to avoid this, but I'm not sure what is standard for VCF specifications. I plan to copy that VCF file into the VCF testdata and test it along with the new tests here to handle a case where there is no format or sample data.

Reply:
Could you do format[0] or format[1] ==  "." for situations where a space is added before the "."?

Reply:
i don't know vcf that well, but we typically set up formatting as tab-based so a space followed directly by a period wouldn't be read as separate fields in the reader. format[1] would point to the next field (the field directly **after** the first tab in the file) if you go to func processVcfLine in vcf.go you can see that the lines of the vcf files are split on the tab (line 90) and then it identifies the fields based on which parts of the file is being processed (so data[0] which is the info directly **before** the first tab is always processed as the chr number)